tests/unit_test/channels/whatsapp_broadcast_test.py

```
@@ -565,4 +566,131 @@ def test_log_failed_messages():
+
+        status_flag, status_code, response = await whatsapp_broadcast.send_template_message(template_id, recipient, language_code, components, namespace, flowname)
+
+        assert status_flag is True
+        assert status_code == 200
+        assert response == {}
+
+        mock_send_template_message_async.assert_called_once_with(template_id, recipient, language_code, components, namespace)
+        mock_add_event_log.assert_called_once()
+
+@pytest.mark.asyncio
+async def test_send_template_message_success_with_flow():
```

Ruff (0.8.2)

Redefinition of unused `test_send_template_message_success_with_flow` from line 614

(F811)